changelog/_1576.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I don't think you would need this file - just the 10285 would suffice.

Reply:
Fixed.